spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -130,14 +155,9 @@ function (k Keeper) DeleteActiveChannelID(portID string) {
 
 To register an interchain account we require an off-chain process (relayer) to listen for `OnChannelOpenInit` events with the capability to finish a channel creation handshake on a given connection. 
 
-1. The controller chain binds a new IBC port with an id composed of the *source/counterparty connection-ids* & the *interchain account owner address*
+1. The controller chain binds a new IBC port with the controller portID for a given *source/counterparty connection-ids* and *interchain account owner address*.
 
-The IBC port identifier will look like this:
-```
-ics27-1.{connection-id}.{counterparty-connection-id}.{owner-account-address}
-```
-
-This port will be used to create channels between the controller & host chain for a specific owner/interchain account pair. Only the account with `{owner-account-address}` matching the bound port will be authorized to send IBC packets over channels created with `ics27-1.{connection-id}.{counterparty-connection-id}.{owner-account-address}`. It is up to each controller chain to enforce this port registration and access on the controller side. 
+This port will be used to create channels between the controller & host chain for a specific owner/interchain account pair. Only the account with `{owner-account-address}` matching the bound port will be authorized to send IBC packets over channels created with the controller portID. It is up to each controller chain to enforce this port registration and access on the controller side. 
```

Comment:
When a channel is reopen it uses the same port ids  (as the previous one) and a new channel-id  ?

Reply:
Correct. Though this is a completely new channel